conftest.py

```diff
@@ -104,3 +106,21 @@ def test_post_teardown():
 
     # Make sure we revert everything we might have changed to prefixers.
     amo.urlresolvers.clean_url_prefixes()
+
+
+@pytest.fixture
+def admin_group(db):
```

Comment:
Oh gosh. Is this py.test magic because the test function takes a `mozilla_user`?